### PR TITLE
Add fix for prop 597 code issue

### PR DESCRIPTION
**Pull request by ValarDragon**
Proposal 597 broke the cfmm-gamm pool links, due to the governance logic not behaving as expected (replacing a list of instead of fixing it)

Proposed for this to be included in the v19 upgrade due to urgency at hand.

### CODE REVIEW

**Comment by ValarDragon** [comment]
devbot add changelog fix prop-572 introduced issue

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Misc Improvements
+
+* [#6279](https://github.com/osmosis-labs/osmosis/pull/6279) fix prop-572 introduced issue
```
Comment (p0mvn) [inline CHANGELOG.md:47]:
```suggestion
* [#6279](https://github.com/osmosis-labs/osmosis/pull/6279) fix prop-597 introduced issue
```

File: app/upgrades/v19/constants.go [hunk 2]
```diff
@@ -17,3 +18,40 @@ var Upgrade = upgrades.Upgrade{
...
+	{BalancerPoolId: 573, ClPoolId: 1109},  // BTSG
+	{BalancerPoolId: 641, ClPoolId: 1110},  // UMEE
+	{BalancerPoolId: 605, ClPoolId: 1111},  // HUAHUA
+	{BalancerPoolId: 971, ClPoolId: 1112},  // NCT
+	{BalancerPoolId: 625, ClPoolId: 1113},  // GRAV
+	// created at https://www.mintscan.io/osmosis/proposals/597
+	{BalancerPoolId: 678, ClPoolId: 1133}, // USDC
+	{BalancerPoolId: 704, ClPoolId: 1134}, // WETH
+	{BalancerPoolId: 1, ClPoolId: 1135},   // ATOM
+	{BalancerPoolId: 803, ClPoolId: 1136}, // stATOM
```
Comment (p0mvn) [inline app/upgrades/v19/constants.go:56]: ACK pool IDs

File: app/upgrades/v19/constants.go [hunk 3]
```diff
@@ -17,3 +18,40 @@ var Upgrade = upgrades.Upgrade{
...
 }
+
+var records = []migration.BalancerToConcentratedPoolLink{
+	// created at v16
+	{BalancerPoolId: 674, ClPoolId: 1066}, // DAI
+	// created at https://www.mintscan.io/osmosis/proposals/571
+	{BalancerPoolId: 837, ClPoolId: 1088},  // IST
+	{BalancerPoolId: 857, ClPoolId: 1089},  // CMST
+	{BalancerPoolId: 712, ClPoolId: 1090},  // WBTC
+	{BalancerPoolId: 773, ClPoolId: 1091},  // DOT
```
Comment (p0mvn) [inline app/upgrades/v19/constants.go:29]: ACK

File: app/upgrades/v19/constants.go [hunk 4]
```diff
@@ -17,3 +18,40 @@ var Upgrade = upgrades.Upgrade{
...
+	{BalancerPoolId: 9, ClPoolId: 1092},    // CRO
+	{BalancerPoolId: 3, ClPoolId: 1093},    // AKT
+	{BalancerPoolId: 812, ClPoolId: 1094},  // AXL
+	{BalancerPoolId: 584, ClPoolId: 1095},  // SCRT
+	{BalancerPoolId: 604, ClPoolId: 1096},  // STARS
+	{BalancerPoolId: 497, ClPoolId: 1097},  // JUNO
```
Comment (p0mvn) [inline app/upgrades/v19/constants.go:35]: ACK pool IDs

**Review by p0mvn — APPROVED** [review]
Validated that all pool IDs from linked proposals are present

Corrected the prop id that broke it to 597

**Comment by p0mvn** [comment]
e2e is failing because init genesis is not updated accordingly.

Stated to update it in: https://github.com/osmosis-labs/osmosis/pull/6280

However, I think we should merge as is and instead just test on a state exported testnet

**Pull request merged**